src/components/views/onboarding/permissions.tsx

```diff
@@ -49,7 +49,7 @@ export const Permissions: FC<any> = () => {
     try {
       app.showActivityIndicator();
 
-      await app.clearContext();
+      await app.clearContext({loading: true});
```

Comment:
I considered that but clearContext is async so there would be a delay before the activity indicator appears, which could make the user think the button hasn't worked and cause them to double-press

Reply:
I don't believe this should exist here at all - maybe an overhang from Leave not clearing all data correctly previously?

Reply:
This clearContext has been in since https://github.com/project-vagabond/covid-green-app/commit/560038dd32f537feb1c8e6ca20711f387d29752f 29 days ago

The problem appears to be cause by https://github.com/project-vagabond/covid-green-app/commit/ce0f1b5c051a734262225a871a94f8c1fa0f03ac nuking the state in clearContext instead of selectively deleting individual items

Reply:
I'm also opting for getting rid of it. @AlanSl

Reply:
When a user leaves - all data should be removed - so nuking is appropriate.

+1 to remove it

Reply:
If you're both 100% sure there's no side effects, I'll do that